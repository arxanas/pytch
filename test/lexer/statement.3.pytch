let foo =
  let bar =
    let baz = 1
    baz
foo
