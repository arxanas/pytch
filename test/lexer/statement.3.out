'let'
leading whitespace ' '
identifier 'foo'
leading whitespace ' '
'='
trailing newline '\n'
leading whitespace '  '
'let'
leading whitespace ' '
identifier 'bar'
leading whitespace ' '
'='
trailing newline '\n'
leading whitespace '    '
'let'
leading whitespace ' '
identifier 'baz'
leading whitespace ' '
'='
leading whitespace ' '
integer literal '1'
trailing newline '\n'
the end of a 'let' binding
leading whitespace '    '
identifier 'baz'
trailing newline '\n'
the end of a 'let' binding
the end of a 'let' binding
identifier 'foo'
trailing newline '\n'
the end of the file
